aimmdb/ingest/aimmdb_backup.py

```diff
@@ -0,0 +1,102 @@
+import json
+import os
+import pandas as pd
+
+def download_aimmdd_data(c):
+    """
+    Navigate through a tiled client and download all the entries that are found
+
+    Parameters
+    ----------
+    c : tiled.client
+
+    Returns
+    -------
+    None.
+
+    """
+    
+    if not os.path.exists("files/"):
```

Comment:
There is an optional argument to say "Create the directory only if it doesn't already exist." https://docs.python.org/3/library/os.html#os.makedirs